### PR TITLE
Scala 2.12.19 draft release notes

### PR DESCRIPTION
**Pull request by SethTisue**
feedback welcome

### CODE REVIEW

File: releases/2.12.19.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+## Scala 2.12 notes
+
+The [release notes for Scala 2.12.0](https://github.com/scala/scala/releases/v2.12.0) have important information applicable to the whole 2.12 series.
+
+## Obtaining Scala
+
+Scala releases are available through a variety of channels, including (but not limited to):
+
+* Bump the `scalaVersion` setting in your sbt project
+* Bump the `using scala` setting in your Scala-CLI project
```
Comment (Philippus) [inline releases/2.12.19.md]:
```suggestion
* Bump the `using scala` setting in your Scala CLI project
```

**Review by Philippus — COMMENTED** [review]
It says "Scala CLI" on the website.

**Comment by som-snytt** [comment]
It should be `CLI: Scala` because `CSI: Miami`, `CSI: Vegas` etc.

**Comment by som-snytt** [comment]
That is aside from `Law & Order: SBT`.

**Comment by SethTisue** [comment]
These notes are now at https://github.com/scala/scala/releases/tag/v2.12.19 . But they can still be modified; feel free to continue to use this PR to suggest improvements.

**Pull request closed without merging**